setup.cfg

```diff
@@ -33,6 +33,7 @@ install_requires =
     django>=3.1.0
     django-better-admin-arrayfield
     django-solo
+    django-choices
```

Comment:
let's just use django's built in enums instead of django-choices for this: https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types